lib/web_ui/dev/steps/run_tests_step.dart

```diff
@@ -130,11 +130,11 @@ class RunTestsStep implements PipelineStep {
       _checkExitCode('Unit tests');
     }
 
-    // Run screenshot tests one at a time. Otherwise, tests end up
-    // screenshotting each other.
-    for (final FilePath screenshotTestFilePath in screenshotTestFiles) {
+    // Run screenshot tests one at a time to prevent tests from screenshotting
```

Comment:
This comment doesn't apply anymore, does it?

Reply:
It does. We set `concurrency: 1` for screenshots, and `concurrency: _testConcurrency` for non-screenshots. What we did previously was _in addition_ to `concurrency` we also ran in separate batches, which causes the browser to be restarted with each batch.